plugin/storage/factory.go

```diff
@@ -139,6 +160,21 @@ func (f *Factory) AddFlags(flagSet *flag.FlagSet) {
 			conf.AddFlags(flagSet)
 		}
 	}
+	addFlags(flagSet)
+}
+
+// addFlags add flags for factory
+func addFlags(flagSet *flag.FlagSet) {
+	flagSet.Float64(
+		downsamplingRatio,
+		defaultDownsamplingRatio,
+		"Downsampling ratio defines the ratio of spans for downsampling. Number between 0 ~ 1.0. Default is 1.0 with no downsampling. Values not in the range of 0 ~ 1.0 will be set to default. e.g ratio = 0.3 means we are keeping 30% of spans and dropping 70% of spans.",
+	)
+	flagSet.String(
+		downsamplingHashSalt,
+		defaultDownsamplingHashSalt,
+		"downsamplingHashSalt defines the hash salt for downsampling. Default is empty string.",
```

Comment:
simplify comment (viper shows default automatically, no need to explain):

Salt used when hashing trace id for downsampling.